tests/e2e/features/3DViewer.feature

```diff
@@ -0,0 +1,40 @@
+Feature: preview 3D model
+    As a user
+    I want to preview a 3D model
+    So that I can make sure this is the correct file (e.g. before sharing it with others)
+
+
+    Scenario: preview 3D model
+        Given the user has logged in with username "admin" and password "admin"
+        And the user has uploaded the following 3D models:
+            | model1.glb |
+            | model2.glb |
+
+        # preview 3D model
+        When the user opens the file "model1.glb" in the 3D model viewer
+        Then the 3D model will be display in the viewport
```

Comment:
usually we describe what should happen, not what happens

> Then steps describe what should be the case if the When step(s) happened successfully. They should contain the word should somewhere in the step text.

https://doc.owncloud.com/server/next/developer_manual/testing/acceptance-tests.html#writing-a-then-step

Reply:
Thanks, I've updated the scenario

Reply:
some questions
- should we use text that is on UI labels at all in scenarios or just describe the action since label text might change or be different if the UI is set to a different language...
- should the manipulation mode be specified (e.g. using the mouse)?

Reply:
I personally would not use the labels/texts in the scenarios but describe the functionality the user wants to achieve. But there are other practices out there, so maybe for @saw-jan to say what he wants

If there is currently only one mode, don't mention it. One day there will be an other mode and then add the tests and refactor old one. [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)

Reply:
I agree with Artur

Reply:
Thanks for input @saw-jan , I've updated the feature file accordingly